### PR TITLE
Added documentation #35

### PR DESCRIPTION
**Pull request by kailashchoudhary11**

### CODE REVIEW

**Comment by Agent-Hellboy** [comment]
please, add a description describing how to do that and how can someone build the doc and open it in the browser.

**Comment by kailashchoudhary11** [comment]
I think their [official documentation]  (https://www.sphinx-doc.org/en/master/tutorial/index.html) covers everything

**Comment by kailashchoudhary11** [comment]
I think their [official documentation](https://www.sphinx-doc.org/en/master/tutorial/index.html) explains everything!

**Comment by kailashchoudhary11** [comment]
The html files are included in docs/build/html/ directory. And if one want to make changes they can make changes in the .rst files included in the docs/source/ directory and then run `make html` in the docs/ directory.

**Comment by Agent-Hellboy** [comment]
You should add a new section as documentation in README, you can remove the example section from README

**Comment by kailashchoudhary11** [comment]
> You should add a new section as documentation in README, you can remove the example section from README

Okay Sure!

**Comment by kailashchoudhary11** [comment]
> You should add a new section as documentation in README, you can remove the example section from README

You mean i should remove all the codes from the Readme?

**Comment by Agent-Hellboy** [comment]
> You mean i should remove all the codes from the Readme?

Yes, except the first example remove all.

**Comment by kailashchoudhary11** [comment]
> > You mean i should remove all the codes from the Readme?
> 
> Yes, except the first example remove all.

Please review!

**Comment by Agent-Hellboy** [comment]
> Please review!

Thanks, I will review on weekends

**Comment by kailashchoudhary11** [comment]
@Agent-Hellboy Any updates?

File: README.rst [hunk 1]
```diff
@@ -61,217 +59,6 @@ using
    >>> l.get_title()
    'Manoj Bajpayee Vs. Atul Khatri | Epic Middle class Face-off | The Family Man | Amazon Prime Video'
    l is a VidInfo object
-.. code-block:: python
-
-
```
Comment (Agent-Hellboy) [inline README.rst:66]: keep this in README

File: README.rst [hunk 2]
```diff
@@ -61,217 +59,6 @@ using
...
-    >>> vid
-    [<util.vidinfo.VidInfo object at 0x7ff971539e10>]
-
-
-    #You can fire dir on this object to get the attribute and method of the object.
-    >>> dir(vid[0])
-    ['_Info__API_KEY', '__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', '_id', 'channel_info', 'get_description', 'get_image_url', 'get_link', 'get_publishedtime', 'get_title', 'keyword', 'maxlen', 'open_id', 'order', 'result', 'type', 'youtube']
-
-    # To get the description of the video
-    >>> vid[0].get_description()
```
Comment (Agent-Hellboy) [inline README.rst:79]: this one also and after that for other APIs, you can tell people to check doc.

**Review by Agent-Hellboy — CHANGES_REQUESTED** [review]
include files that are required to generate HTML and other files, no need to include the files generated by a sphinx, you can add instructions to generate HTML and other files.

**Comment by Agent-Hellboy** [comment]
>@Agent-Hellboy Any updates?

added few comments, please check

**Comment by kailashchoudhary11** [comment]
> > @Agent-Hellboy Any updates?
> 
> added few comments, please check

Okay Sure!

**Comment by kailashchoudhary11** [comment]
Please Review @Agent-Hellboy

**Comment by Agent-Hellboy** [comment]
Thanks for the PR @kailashchoudhary11

**Pull request merged**